static/js/billing/billing.js

```diff
@@ -43,8 +43,15 @@ $(function () {
         e.preventDefault();
     });
 
-    $('#billing-tabs a').click(function (e) {
-        e.preventDefault();
+    var hash = window.location.hash;
+    if (hash) {
+        $('ul.nav a[href="' + hash + '"]').tab('show');
```

Comment:
Adding an outer `#billing-tabs` around some of these selectors would make it a lot more robust to any future tabs we might add.  Otherwise lgtm.